packages/components/switch/src/switch.tsx

```diff
@@ -116,6 +116,8 @@ export const Switch = forwardRef<SwitchProps, "input">((props, ref) => {
       {...rest}
     >
       <ui.input
+        role="switch"
+        aria-checked={isChecked}
```

Comment:
Shouldn't this also be in `getInputProps`?

Reply:
@108yen 

If we do that, I'd like to create `useSwitch` based on `useCheckbox`, and add `role` and `aria-checked` there. What do you think?

Reply:
@koralle 
Sorry. I missed that you are using `useCheckbox`.
Is it possible to put it in `useCheckbox` if only `aria-checked={isChecked}`?

Reply:
@108yen 

In the `Checkbox` component, the `aria-checked` attribute can have an intermediate value (`mixed`). On the other hand, this does not exist in the `Switch` component. Therefore, I don't think it's appropriate to apply `useCheckbox`, which includes the declaration of the `aria-checked` attribute, to the `Switch` component.

> * When checked, the checkbox element has state [aria-checked](https://w3c.github.io/aria/#aria-checked) set to `true`.
> * When not checked, it has state [aria-checked](https://w3c.github.io/aria/#aria-checked) set to `false`.
> * When partially checked, it has state [aria-checked](https://w3c.github.io/aria/#aria-checked) set to `mixed`.

https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/

Reply:
Thanks for the reply.
This seems to be the only way to do it this time. Approved.